### ISSUE TITLE
Beacons

### ISSUE THREAD

**Issue by Gerold55** [Wuzzy2/MineClone2-Bugs#292]
Can this work for the model of the beacon?
https://1drv.ms/u/s!AgaDE7LeGFbL7n7VpPNTtkdg1sFS

**Comment by kingoscargames** [comment]
I dont think this needs be added yet becase the nether star isnt obtaianable in survival yet(i believe).

**Comment by kingoscargames** [comment]
The wiki says that it is a Block Entity not a block.

**Comment by Gerold55** [comment]
Yeah there's no wither boss yet because of no nether I suggested this mod: https://forum.minetest.net/viewtopic.php?f=11&t=16009&hilit=Nether
Then the only thing needs 2 be done is to create the separate dimensions

**Comment by Gerold55** [comment]
That's what this mod does reading further down in the comments of the mod

**Comment by Gerold55** [comment]
![screenshot 34](https://user-images.githubusercontent.com/13843095/27354800-d238cd62-55d6-11e7-9d40-20538feb378a.png)

**Comment by kingoscargames** [comment]
I meant under the bedrock of the world not generating its own bedrock

**Comment by kingoscargames** [comment]
Maybe there could be a temporary recipe for the nether star (maybe 4 diamond blcoks and 1 gold block).

**Comment by Gerold55** [comment]
Yeah I think that it generates the dimension somewhere in the air

**Comment by kingoscargames** [comment]
That could be changed (dont know how though i am not a real coder or something but know a little bit about it).

**Comment by kingoscargames** [comment]
Maybe this:https://forum.minetest.net/viewtopic.php?f=11&t=5790&hilit=Nether+pilzadam ?

**Comment by kingoscargames** [comment]
Or this? : https://forum.minetest.net/viewtopic.php?f=9&t=15893&hilit=Nether+pilzadam

**Comment by Gerold55** [comment]
It could work but would need some major edits like a bigger nether and nether fortresses and pools of lava

**Comment by Wuzzy2** [comment]
Items being unobtainable will not prevent me from adding them early. There's still Creative Mode, after all.

The Nether and the End haven't been added yet because the map generators need to be implemented first. All Nether mods I have seen so far have pretty poor map generation or they are just too different from the MC Nether.


Also, finishing the Overworld biomes has obviously a higher priority. The Overworld is still the most important of the three “dimensions”.

**Comment by Gerold55** [comment]
@Wuzzy are you able to eventually add the beacon and are you able to use the model I made?
